handlers/commands/commands.js

```diff
@@ -22,27 +29,52 @@ const commandReference = `\
 <code>/user</code> - Shows user's status and warns.
 <code>/addcommand &lt;name&gt;</code> - to create a custom command.
 <code>/removecommand &lt;name&gt;</code> - to remove a custom command.
-
+`;
+const userCommands = `\
 <b>Commands for everyone</b>:
 <code>/staff</code> - Shows a list of admins.
 <code>/link</code> - Show the current group's link.
 <code>/groups</code> - Show a list of groups which the bot is admin in.
 <code>/report</code> - Reports the replied-to message to admins.
 `;
```

Comment:
Could add an extra newline at the end of every string (and remove the newline from L71), so that there is space between groups of builitin commands.

Reply:
Done